src/index.ts

```diff
@@ -46,12 +46,14 @@ export type TransactionOptions = {
 /**
  * Launch Puppeteer chromium instance with MetaMask plugin installed
  * */
-export async function launch(
-  puppeteerLib: typeof puppeteer,
-  { args, metamaskLocation, ...rest }: LaunchOptions = {},
-): Promise<puppeteer.Browser> {
+export async function launch(puppeteerLib: typeof puppeteer, options: LaunchOptions): Promise<puppeteer.Browser> {
+  if (!options || !options.metamaskVersion)
```

Comment:
I think we should default to latest version that is stable for us.
In the docs we can just put notice that they can use against latest metamask but there is a chance dappeteer won't work against it.